src/middleware/ErrorHandler.ts

```diff
@@ -2,13 +2,20 @@ import { NextFunction, Request, Response } from 'express';
 import { INTERNAL_SERVER_ERROR } from 'http-status-codes';
 
 // @ts-ignore
-export function handler(err: any, req: Request, res: Response, nextFunction: NextFunction): void {
-    console.error(err.message);
+function notFoundHandler(req: Request, res: Response, next: NextFunction): void {
+    res.status(404).render('error');
+    next()
```

Comment:
Do you need to call `next()` in page not found scenario? I guess rendering error page is everything that will happen is such scenario.